wsl-builder/common/paths.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // getPathWith return the parent directory containing subdirectory. It goes up from
```

Comment:
You need to rename the documentation accordingly `GetPathWith` (and fix my typo `return` -> `returns` as the form should be `ExportedSymbol <verb> …`)